boards/arm64/rcar_h3ulcb_ca57/rcar_h3ulcb_ca57_defconfig

```diff
@@ -25,9 +19,7 @@ CONFIG_MMC_RCA=1
 
 # Enable console
 CONFIG_CONSOLE=y
-CONFIG_UART_RCAR=y
```

Comment:
Don't think that these changes should belong to commit with such description.

Reply:
I have extended commit message, thanks

Reply:
Commit name still says "dts: arm64: changes for r8a7795.dtsi".

Also, please take note that this is too broad for the commit name. Any patch can be described as "changes to some_file.ext"

Reply:
as I have written I changed commit message and @firscity has said that changes isn't related to commit description
nobody has said nothing about commit name

Reply:
Commit name is a part of the commit description as a whole. Just by looking at your commit (for example when doing `git log --oneline`) tells me that you are touching only r8a7795.dtsi. Which is not true.